### ISSUE TITLE
Skin editor default shortcut conflicts with song select previous random (rewind)

### ISSUE THREAD

**Issue by Joehuu** [ppy/osu#12776]
**Describe the bug:** They both use shift + f2.

**osu!lazer version:** master

**Comment by peppy** [comment]
Do you have a proposal for this? Maybe moving skin editing to ctrl+shift+s or something?

**Comment by frenzibyte** [comment]
I'm not sure if the skin editor is intended to run on the beatmap editor screen and otherwise, but Ctrl+Shift+S will then conflict with some saving keybinds there.

I'm thinking more about Ctrl+Alt+S or even Ctrl+Alt+F12 for an overlay that can be opened literally anywhere throughout the game.

**Comment by enoslayd** [comment]
Maybe ctrl+alt+shift+s? Stable uses that for skin reload but since it isnt in lazer maybe use that instead

**Comment by frenzibyte** [comment]
3 modifiers combination sounds really too much just to open a skin editor overlay.

**Comment by peppy** [comment]
Where's the code for the Ctrl+Shift+S binding in the editor?

**Comment by frenzibyte** [comment]
There's none currently, I meant it will conflict if Ctrl+Shift+S gets set to a certain action there later on, which seems likely to happen for beatmap editor.

**Comment by Joehuu** [comment]
> Where's the code for the Ctrl+Shift+S binding in the editor?

If we are still conforming to stable, then https://osu.ppy.sh/wiki/en/Shortcut_key_reference#object-manipulation.

> Do you have a proposal for this?

I would go with Ctrl+Alt+S too. It's one modifier less than the infamous reload shortcut, and doesn't conflict any not-implemented stable shortcuts.

> Ctrl+F(x) if we have any free function keys left?

I believe we should reserve them for more framework/debug shortcuts? Even if they are gonna be disabled for release builds, they will still conflict in dev builds.

Ctrl + F
- 1: draw vis
- 2: global stats
- 3: textures
- 4: toggle mute
- 5:
- 6:
- 7: toggle frame limiter
- 8:
- 9:
- 10: toggle debug logs
- 11: toggle fps counter states
- 12:

**Comment by peppy** [comment]
I think we really need to go through and create a standard for default key bindings.

On a quick check, I'd propose the following changes:

- Remove F1 F3 F5 defaults for music controls
- Remove Ctrl-F4 mute binding
- Remove Ctrl-Shift-A verify screen binding from editor (and move to classic preset if required)

And then making rough standards for what should use what bindings:

- Ctrl-Fkey reserved for framework / system stuff
- Fkey used for switching top level modes of the current game screen
- Ctrl-Shift-alphanumeric used for global game elements (ie. overlay toggles, such as the skin editor)
- Ctrl-alphanumeric used for local elements (ie. beatmap editor manipulation). Can fall back to Alt prefix if we run out.
- Avoid using Ctrl-Alt-anything anywhere. Not a huge fan of this combination for defaults (maybe because it's a bit of a weird combination on macOS).
These could be applied as a "classic" preset on loading the game for the first time in the future.

**Comment by smoogipoo** [comment]
I agree with the standards proposal. Probably should be made into a tracking issue somewhere.
